docs/src/prototypes/customToolbar/index.tsx

```diff
@@ -58,21 +58,53 @@ const CustomToolbarPrototype: React.FunctionComponent = () => {
   const [currentSlide, setCurrentSlide] = React.useState(23)
   const totalSlides = 34
 
+  const [telemetryEnabled] = useBooleanKnob({ name: 'telemetryEnabled', initialValue: true })
+  const telemetryRef = React.useRef<Telemetry>()
+
   let theme = {}
   if (themeName === 'teamsDark') {
     theme = mergeThemes(themes.teamsDark, darkThemeOverrides)
   } else if (themeName === 'teamsHighContrast') {
     theme = mergeThemes(themes.teamsHighContrast, darkThemeOverrides, highContrastThemeOverrides)
   }
 
+  React.useEffect(() => {
```

Comment:
Just an example, but let's leave it here for now, until we will have a dedicated doc.